ggml/src/ggml-sycl/softmax.cpp

```diff
@@ -29,9 +37,10 @@ static void soft_max_f32(const float * x, const float * mask, float * dst, const
         slope = sycl::pow(base, float(exp));
     }
 
-    float *vals = vals_smem ? buf + std::max(nwarps, WARP_SIZE) : dst + rowx * ncols;
+    float *vals = vals_smem ? buf + sycl::max(nwarps, WARP_SIZE) : dst + rowx * ncols;
     float max_val = -INFINITY;
 
+#pragma unroll
```

Comment:
I'm worried about the impact of this on various devices. It would be less controversial to remove it for now and only add the pragma unroll if we can verify that this improves performance overall for some devices with little regressions.

Reply:
I've not measured any difference in performance for the models 70B or 8B on PVC and A100. I would suggest to remove it unless we measure an improvement in one device.

Reply:
Okay.